designer/client/src/components/spinner/Spinner.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Box, CircularProgress, ThemeProvider, circularProgressClasses, createTheme } from "@mui/material";
+
+type Props = {
+    show: boolean;
+};
+
+function LoaderSpinner(props: Props) {
+    return !props.show ? null : <CircularProgressWrapper />;
+}
+
+export default LoaderSpinner;
+
+function CircularProgressWrapper() {
+    const theme = createTheme({});
+    return (
+        <ThemeProvider theme={theme}>
+            <Box sx={{ position: "absolute", left: "50%", top: "50% " }}>
+                <CircularProgress
+                    variant="determinate"
+                    sx={{ color: (theme) => theme.palette.grey[200] }}
```

Comment:
Why do we want this grey color here, can't we just use the same green Nussknacker color? (probably the same as in case of previous spinner)

Reply:
in the previous spinner it use gray color

Reply:
You are right! I was sure that it was the green color 🙈